exercises/practice/pig-latin/.meta/example.cpp

```diff
@@ -0,0 +1,31 @@
+#include "pig_latin.h"
+
+namespace pig_latin {
+std::string translate_word(std::string word) {
+    for (std::string start : {"a", "e", "i", "o", "u", "yt", "xr"}) {
```

Comment:
No need to copy the character (though for one or two characters it probably doesn't really make a difference):
```suggestion
    for (std::string_view start : {"a", "e", "i", "o", "u", "yt", "xr"}) {
```